sim/rogue/subtlety/master_of_subtlety.go

```diff
@@ -0,0 +1,30 @@
+package subtlety
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (subRogue *SubtletyRogue) registerMasterOfSubtletyCD() {
+	var MasterOfSubtletyID = core.ActionID{SpellID: 31223}
+
+	percent := 1.1
+
+	effectDuration := time.Second * 6
+	if subRogue.StealthAura.IsActive() {
+		effectDuration = core.NeverExpires
+	}
```

Comment:
this is always false as during registration you dont have the sim running yet. Whats the idea of thsi exactly?

Reply:
Another unchanged from WotLK implementation. Will need to explore.

Reply:
So for this and Overkill (assassination talent, has this same check), there needs to exist some logic to make it no duration during stealth, and a fixed duration when exiting stealth. I'm going to implement this logic in Stealth.go as it already performs the checks to Activate these auras.